indico/htdocs/js/indico/modules/surveys.js

```diff
@@ -53,6 +53,73 @@
         $('.js-add-question-dropdown').dropdown();
     }
 
+    global.setupSubmissionButtons = function setupSubmissionButtons() {
+        $('#export-submissions').on('click', function(evt) {
+            var $this = $(this);
+
+            var form = $('<form>', {
+                method: 'POST',
+                action: $this.data('href'),
+            });
+
+            $('.submission-ids:checked').each(function() {
+                $('<input>', {type: 'hidden', name: 'submission_ids', value: this.value}).appendTo(form);
+            });
+
+            form.appendTo('body').submit();
+            form.remove();
+        });
+
+        function _disableButtons() {
+            if ($('li.submission-row').length == 0) {
+                $('.js-submission-action').prop('disabled', true);
+            }
+        }
+
+        $('.js-delete-submission').on('indico:confirmed', function(evt) {
+            evt.preventDefault();
+            var $this = $(this);
+
+            $.ajax({
+                url: $this.data('href'),
+                method: $this.data('method'),
+                data: {
+                    submission_ids: $this.data('submission-id')
+                },
+                complete: IndicoUI.Dialogs.Util.progress(),
+                error: handleAjaxError,
+                success: function() {
+                    $this.closest('li.submission-row').remove();
```

Comment:
I'd only put the class - adding the tag name too doesn't provide any real value and may make evaluating the selector slower